docs/graphql/manual/guides/production-checklist.rst

```diff
@@ -0,0 +1,76 @@
+.. _production-checklist:
+
+Production checklist
+====================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+This guide is a checklist for configuring and securing GraphQL engine for a production deployment.
+
+Security
+--------
+
+The following steps are recommended to better secure your production GraphQL engine instance:
+
+Admin secret
+^^^^^^^^^^^^
+
+Configure an admin secret as described :doc:`here <../deployment/securing-graphql-endpoint>`.
+
+Reverse proxy & API gateway
```

Comment:
We can change this section to just say Enable HTTPS. Let's not call it reverse proxy/API gateway. 
And the line can be: "Use a component or a service from your cloud vendor to enable HTTPS". Examples, can be nginx, caddy like they are.